cni/network/network_windows.go

```diff
@@ -248,9 +248,8 @@ func getEndpointDNSSettings(nwCfg *cni.NetworkConfig, result *cniTypesCurr.Resul
 }
 
 // getPoliciesFromRuntimeCfg returns network policies from network config.
-func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig, isIPv6Enabled bool) []policy.Policy {
-	logger.Info("Runtime Info",
-		zap.Any("config", nwCfg.RuntimeConfig))
+func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig, isIPv6Enabled bool) ([]policy.Policy, error) {
```

Comment:
if `isIPv6Enabled` not used, can we remove it?

Reply:
Vipul and I thought if customer can modify ipam configuration to set isIPv6Enabled to true and create windows pod in v4Overlay cluster and it would cause the issue.

Reply:
Since hostIP is not required field, so we still need "isIPv6Enabled" flag to check if it's ipv4 or ipv6 policy.